app/assets/javascripts/discourse/app/components/d-editor.js

```diff
@@ -723,6 +648,8 @@ export default class DEditor extends Component {
       this.element?.removeEventListener("paste", textManipulation.paste);
 
       textManipulation.autocomplete("destroy");
```

Comment:
wonder if we shouln't have a destroy function on TextareaManipulation and maybe if this call shouldn't directly be in teardownUpstream call ?

Reply:
It could be moved to a destroy fn in TextareaManipulation, but it would be the only line in a destructor at least for now

I don't think we should call it upstream/on ComposerEditor. I'm keeping the destructors as close as possible to the constructors, and all autocompletes are being set-up here in DEditor, so destructing them should also be here IMO

Reply:
ok works for me :+1: